src/sbt-test/docker/jdk-versions/build.sbt

```diff
@@ -0,0 +1,31 @@
+val basename = "jdk-versions"
+
+scalacOptions in ThisBuild := Seq("-target:jvm-1.8")
```

Comment:
it seems the scala doc compiler doesn't like this setting. I never actually fought
the docs compiler (only my colleagues), so sorry for not being a greater help here :disappointed: 

IIRC you can either turn the scaladoc generation off or do some research on how
to fix this :sunglasses: I'm fine with either way :+1:

Reply:
Thanks for pointing this out. Will push a fix. Normally limiting the scope should fix this.

Reply:
Awesome, thanks :)